hedy.py

```diff
@@ -643,11 +646,13 @@ def ask(self, args):
         return "answer = input('" + argument + "')"
 
     def forward(self,args):
-        # when a not-number is given, we simply use 50 as default
+        if len(args) == 0:
+            return self.make_forward(50)
+
         try:
             parameter = int(args[0])
         except:
-            parameter = 50
+            raise InvalidArgumentException(command = "forward", arg = args[0])
```

Comment:
Can we loop in allowed types here for a better error message (see screenshots in discussion)